lxd/images.go

```diff
@@ -973,6 +973,7 @@ func imagesPost(d *Daemon, r *http.Request) response.Response {
 
 	projectName := request.ProjectParam(r)
 
+	// If the client is not authenticated, CheckPermission will return false.
```

Comment:
false? You mean an error, it cant return false?

Reply:
Fixed comment.